source/fundamentals/indexes.txt

```diff
@@ -186,14 +200,102 @@ Strings), and ``title`` fields:
 
       Index created: rated_1_genres_1_title_1
 
-The following is an example of a query that would be covered by the index created in the preceding code snippet:
+The following is an example of a query that is covered by the index
+created in the preceding code snippet:
 
 .. literalinclude:: /examples/generated/IndexesTest.snippet.multikey-index-query.kt
    :language: kotlin
 
-Multikey indexes behave differently from non-multikey indexes in terms of query coverage, index bound computation, and
-sort behavior. For a full explanation of multikey indexes, including a discussion of their behavior and limitations,
-refer to the :manual:`Multikey Indexes page </core/index-multikey>` in the MongoDB manual.
+Multikey indexes behave differently from other indexes in terms of query coverage, index bound computation, and
+sort behavior. To learn more about multikey indexes, including a discussion of their behavior and limitations,
+see the :manual:`Multikey Indexes page </core/index-multikey>` in the MongoDB manual.
+
+.. _kotlin-search-indexes:
+
+Atlas Search Indexes
+~~~~~~~~~~~~~~~~~~~~
+
+The Atlas Search feature enables you to perform full-text searches on
+collections hosted on MongoDB Atlas. The indexes specify the behavior of
+the search and which fields to index.
+
+To learn more about MongoDB Atlas Search, see the
+:atlas:`Atlas Search Indexes </atlas-search/atlas-search-overview/#fts-indexes>`
+documentation.
+
+You can call the following methods on a collection to manage your Atlas Search
+indexes:
+
+- ``createSearchIndex()``
+- ``createSearchIndexes()``
+- ``listSearchIndexes()``
+- ``updateSearchIndex()``
+- ``dropSearchIndex()``
+
+.. note::
+
+   The Atlas Search Index management methods run asynchronously. The
```

Comment:
S: Guessing "Atlas Search Index" isn't a proper noun phrase? Would lowercase + hyphenate the compound modifier

```suggestion
   The Atlas Search index-management methods run asynchronously. The
```